projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdf.java

```diff
@@ -0,0 +1,248 @@
+    private static final String lawsuitPath = "/Lawsuits/";                         // General lawsuits dir.
+    private static String templatePath = lawsuitPath + "/Templates/template.xhtml"; // Lawsuit Template's dir & template's path.
+    private static final String lawsuitOutput = lawsuitPath;                        // Lawsuits output dir.
+
+    // Date & Time formats: (Used for files names)
+    // TODO: Check if this should be as part of a "File manager" class.
+    private static final String DATE_TIME_FORMAT = "dd-M-yyyy hh-mm-ss";
+    private static final String DATE_FORMAT = "dd-M-yyyy";
+    private static final String TIME_ZONE = "Asia/Jerusalem";
+    private static SimpleDateFormat dateAndTimeFormatter;
+    private static SimpleDateFormat dateFormatter;
+
+    // Template's inner text (been read on create from template's file):
+    private static String templateContent = "";
+
+// ------------------------------- Lawsuit Form Fields ------------------------------- //
+//  TODO: Check if preferable to move this to an external file holding constant String values of lawsuit form.
+    private String spamType = "הודעה אלקטרונית";        // Type: SMS/Phone call/Email.
+    private String claimAmount = "סכום תביעה";
+    private Date receivedSpamDate;                    // Date received the spam.
+    private Date currentDate;                         // Current date of lawsuit creation (used for lawsuit signature).
+    private boolean sentHaser = true;                 // Indicates whether user has sent "haser" or not.
+    private boolean sentMoreThanFiveLawsuits = false; // Indicates whether user has sent more then 5 lawsuits already in the past year or not.
+    private String moreThanFiveLawsuits = sentMoreThanFiveLawsuits ? "הגיש" : "לא הגיש/ו";
+    private String claimCase = sentHaser ? "לאחר שהתובע חזר בו מהסכמתו לקבלת דבר/י פרסומת מהנתבע/ים, על-ידי משלוח הודעת סירוב כהגדרתה בחוק" : "למרות שח\"מ לא נתן את הסכמתו המפורשת מראש לקבלת דבר/י הפרסומת";
+
+    // User:
+    private String userPrivateName = "שם פרטי";
+    private String userLastname = "שם משפחה";
+    private String userID = "תז משתמש";
+    private String userAddress = "כתובת משתמש";
+    private String userPhone = "טלפון משתמש";
+    private String userFax = "פקס משתמש";
+
+    // First spam company's details:
+    private String companyName = "שם החברה";
+    private String companyId = "מספר ח.פ";
+    private String companyAddress = "כתובת החברה";
+    private String companyPhone = "טלפון החברה";
+    private String companyFax = "פקס חברה";
+
+    // Second spam company's details:
+    private String company2Name = "שם החברה 2";
+    private String company2Id = "מספר ח.פ 2";
+    private String company2Address = "כתובת החברה 2";
+    private String company2Phone = "טלפון החברה 2";
+    private String company2Fax = "פקס חברה 2";
+// -------------------------------------------------------------------------------------:
+    Button btnCreate;
```

Comment:
We prefer to use full names instead of shortened ones. So I suggest, `s/btnCreate/createButton`.